doc/content/integrations/pubsub/mqtt-client/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "MQTT Client"
+description: ""
+weight: 1
+---
+
+This section explains how to configure a Pub/Sub Integration using the built in MQTT client.
+
+<!--more-->
+
+Creating a Pub/Sub Integration with the built in MQTT client requires you to have an MQTT server running with an available endpoint. [Mosquitto](https://mosquitto.org/) is a popular open source MQTT Server you may use, but {{% tts %}} MQTT client works with any MQTT server.
+
+>No API key is needed for Pub/Sub messaging. All messages received by the Application Server are assumed to be authorized, so be sure to configure appropriate security on your MQTT server.
+
+In your application select the **Pub-subs** submenu from the **Integrations** side menu. Clicking on the **+ Add pub-sub** button will open the Pub/Sub creation screen.
+
+Give your Pub/Sub an **ID**, choose a **Message Format**, and enter a **Base Topic** to subscribe or publish to.
+
+Select **MQTT** as provider.
+
+{{< figure src="pubsub-id.png" alt="Pub/Sub creation screen" >}}
+
+If using Client Certificates, enable **Secure**. You will be asked to upload the **Server Root CA Certificate**, the **Client Certificate**, and the **Client Private Key.**
```

Comment:
"Client Certificates" => "TLS mutual authentication"

I see that **Client Private Key** has sentence casing in the Console; can you fix that as part of this PR?

While you're at it, you might want to change it to "TLS Server Root CA Certificate", "TLS Client Certificate" and "TLS Private Key"

Reply:
Console labels are all sentence casing per design guidelines https://github.com/TheThingsIndustries/lorawan-stack/issues/2109#issuecomment-627690981

Reply:
They are title case, except this one:

<img width="492" alt="Screenshot 2020-05-14 at 11 59 03" src="https://user-images.githubusercontent.com/13334001/81921076-58983280-95da-11ea-8a50-e8c3d5b28730.png">

Reply:
hmm maybe enterprise hasn't been updated? open source looks like this

<img width="638" alt="Screenshot 2020-05-12 at 17 36 07" src="https://user-images.githubusercontent.com/6963436/81714397-4f428500-9477-11ea-969e-7b6852134950.png">
<img width="647" alt="Screenshot 2020-05-12 at 17 36 14" src="https://user-images.githubusercontent.com/6963436/81714404-523d7580-9477-11ea-9631-37d93720b978.png">
<img width="326" alt="Screenshot 2020-05-12 at 17 37 08" src="https://user-images.githubusercontent.com/6963436/81714406-52d60c00-9477-11ea-8662-f946fd9fd9be.png">

Reply:
Ah ok, then it's fine!

Reply:
Still, change the names of the labels